mp/src/game/server/momentum/mom_system_saveloc.cpp

```diff
@@ -159,7 +159,10 @@ void SavedLocation_t::Teleport(CMomentumPlayer* pPlayer)
         pPlayer->SetGravity(m_fGravityScale);
 
     if ( m_savedComponents & SAVELOC_DISABLED_BTNS )
+    {
+        pPlayer->m_afButtonDisabled = 0;
```

Comment:
You could honestly just set the disabled buttons to the ones stored in the saveloc

Reply:
Much simpler! Oops